lib/reports/auditing_report.rb

```diff
@@ -116,15 +116,12 @@ def generate(document)
         header << "Documents"
         header << "Fields Modified"
         header << "Date"
-        header << "Quantity of Each Service Completed"
-        header << "What was the Charge"
-        header << "Who was the Payer"
 
         csv << header
 
         protocols.each do |protocol|
           protocol.line_items.each do |line_item|
-            next unless line_item.versions.where(event: ["create", "update"], created_at: @start_date..@end_date).any? && line_item.service.one_time_fee?
+            next unless line_item.versions.where(event: ["create", "update"], created_at: @start_date..@end_date).any?
```

Comment:
Doesn't this just grab all line_items?  It should be filtering out the non-one_time_fee services

Reply:
This pull request is just to revert my changes. The story needs to be tweaked and is no longer included in this release.